requirements.txt

```diff
@@ -4,3 +4,4 @@ requests==2.20.0
 selenium==3.141.0
 beautifulsoup4==4.7.1
 html5lib==1.0.1
+config==0.4.2
```

Comment:
hello, Im new to using this program, where can I find this document in my system?
kind regards

Reply:
@alfaher You might be looking at `ls -lha /usr/lib/python* | grep config`